proj/src/main/webapp/signup.js

```diff
@@ -31,7 +32,12 @@ function initSignUpButtons(){
                 lat = 999;
                 long = 999;
             }
-            await signUp(fname.value, lname.value, txtEmail.value, txtPassword.value, tagStr.value, lat, long);
+            try {
+                await signUp(fname.value, lname.value, txtEmail.value, txtPassword.value, tagStr.value, lat, long);
+            }
+            catch (e) {
+                alert(e.message);
+            }
```

Comment:
Since there is already a catch within signUp (about lines 81-84ish), is this still needed?

Reply:
That alert doesn't show weak passwords and other registration issues

Reply:
Oh I see, in that case, can you instead put on line 84ish a catch instead of this here? The. way this is now, if there is an issue with splitting tags or something, this will hide it. I want the catch to be constrained to the actual user creation (lines 54-84)
something like `await auth.createUserWithEmailAndPassword(email, pass).then(async function(){... 
complicated logic for setting up chats ...}).catch(function(err){alert(err);});`